src/commands/sites/sites-create-template.js

```diff
@@ -135,14 +135,16 @@ const sitesCreateTemplate = async (repository, options, command) => {
           )
         }
       } else {
+        const { default_branch: branch, full_name: repo, private } = await getRepo(repoResp.full_name, ghToken)
```

Comment:
Could this potentially still fail or does it need a retry?
I'm just wondering whether we're guaranteed to have the repo created in a timely enough manner or whether we could get a 404 here.

Reply:
That makes sense @nickytonline - thanks for the explanation.